experiment/build/generate_cloudbuild.py

```diff
@@ -13,28 +13,69 @@
 # limitations under the License.
 """Generates Cloud Build specification"""
 
-import argparse
-import json
-import yaml
+import os
 
 from experiment.build import docker_images
 
+BASE_TAG = 'gcr.io/fuzzbench'
+EXPERIMENT_TAG = "${_REPO}"
+FUZZER_VAR = "${_FUZZER}"
+BENCHMARK_VAR = "${_BENCHMARK}"
+EXPERIMENT_VAR = "${_EXPERIMENT}"
 
-# TODO(Tanq16): Add unit test for this.
-def create_cloud_build_spec(buildable_images, docker_registry):
-    """Returns Cloud Build specificatiion."""
+
+def _identity(name):
+    return name.replace(FUZZER_VAR, "fuzzer").replace(BENCHMARK_VAR,
+                                                      "benchmark")
```

Comment:
Another thought on this function, aren't we substituting the name in get_images_to_build?

Reply:
Yes, this is only for the ID field of the steps. Basically to keep id as `fuzzer-benchmark-builder` instead of `afl-zlib-builder`. If this abstraction is not required, I'll remove this. Is that fine?